content/ruby/concepts/ruby-on-rails/ruby-on-rails.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: 'Ruby on Rails'
+Description: 'Ruby on Rails is an open-source, server-side Ruby framework used for building web applications.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Browsers'
+  - 'CRUD'
+  - 'Development'
+  - 'Frameworks'
+  - 'Models'
+  - 'Rails'
+  - 'Views'
+CatalogContent:
+  - 'learn-rails'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+<link rel="canonical" href="https://www.codecademy.com/resources/blog/what-is-ruby-on-rails/" />
+
+[**Ruby on Rails**](https://rubyonrails.org/), or "Rails", is an open-source server-side framework written in [Ruby](https://www.codecademy.com/resources/docs/ruby) and used for building web applications. Rails provides pre-built, boilerplate code that can be reused or customized in an application.
+
+While particularly popular with startups, some well-known companies that use Rails include Airbnb, Basecamp, [GitHub](https://www.codecademy.com/resources/docs/general/github), Shopify, and SoundCloud.
+
+## Installation
+
+Rails can be installed as a [Ruby gem](https://www.codecademy.com/resources/docs/ruby/gems). However, the following must be installed beforehand:
+
```

Comment:
Since these need to be installed beforehand, do you think it would be wise to link out "SQLite3" and "Yarn"?

Reply:
Linked!